src/buildParameters.ts

```diff
@@ -22,149 +20,147 @@ export function buildParameters(uri: Uri): EvaluationParameters {
   const libraryDirectory = Utils.dirname(uri);
   const libraryName = Utils.basename(uri).replace('.cql', '').split('-')[0];
   const projectPath = workspace.getWorkspaceFolder(uri)!.uri;
-
-  // todo: make this a setting
-  let terminologyPath: Uri = Utils.resolvePath(projectPath, 'input', 'vocabulary', 'valueset');
-
-  let fhirVersion = getFhirVersion();
-  if (!fhirVersion) {
-    fhirVersion = 'R4';
-    window.showInformationMessage('Unable to determine version of FHIR used. Defaulting to R4.');
-  }
-
+  const terminologyPath: Uri = Utils.resolvePath(projectPath, 'input', 'vocabulary', 'valueset');
+  const fhirVersion = getFhirVersion();
   const optionsPath = Utils.resolvePath(libraryDirectory, 'cql-options.json');
   const measurementPeriod = '';
   const testPath = Utils.resolvePath(projectPath, 'input', 'tests');
   const resultPath = Utils.resolvePath(testPath, 'results');
   const outputPath = Utils.resolvePath(resultPath, `${libraryName}.txt`);
 
   fse.ensureFileSync(outputPath.fsPath);
+  
+  const connectionManager = mockConnectionManager();
```

Comment:
Since we're merging this into the feature branch and there's a chance we could miss it and this could be accidentally merged into the main branch, please add a popup or some other warning that this is running off a mock.

Reply:
removed connection manager

Reply:
Mock to be removed before merge as discussed.